editor/property_editor.cpp

```diff
@@ -3571,8 +3600,8 @@ void PropertyEditor::_draw_transparency(Object *t, const Rect2 &p_rect) {
 
 void PropertyEditor::_item_folded(Object *item_obj) {
 
-	if (updating_folding)
-		return;
+	//if (updating_folding)
```

Comment:
Please remove dead code.

Reply:
And `updating_folding` is unused, so it can be removed everywhere where you define it.